drivers/network/tcpip/include/icmp.h

```diff
@@ -8,10 +8,15 @@
 #pragma once
 
 typedef struct ICMP_HEADER {
-    UCHAR Type;      /* ICMP message type */
-    UCHAR Code;      /* ICMP message code */
-    USHORT Checksum; /* ICMP message checksum */
-    ULONG Unused;    /* ICMP unused */
+    UINT8 Type;      /* ICMP message type */
+    UINT8 Code;      /* ICMP message code */
+    UINT16 Checksum; /* ICMP message checksum */
+    // union {
```

Comment:
There is technically no difference between these. What is the reason for having it and commenting it out? A comment about the source of this definition and the reason for this would be nice.